README.md

```
@@ -1,337 +1,67 @@
+**(Aqui você pode inserir os GIFs ou vídeos do seu projeto, demonstrando as funcionalidades.)**
 
-    class DetailsPage {
-     static const routeName = '/details';
-    }
+* **Listagem de Personagens e Rolagem Infinita**
 
-## Tipos de variáveis
+    ![Rolagem Infinita](https://github.com/user-attachments/assets/6e44268e-cf03-40b8-ad37-b17c0acedd8e)
 
-### Números:
 
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)